panther_analysis_tool/backend/client.py

```diff
@@ -182,3 +182,7 @@ def panthersdk_bulk_upload(
         self, params: PantherSDKBulkUploadParams
     ) -> BackendResponse[PantherSDKBulkUploadResponse]:
         pass
+
+
+def backend_response_failed(resp: BackendResponse) -> bool:
+    return resp.status_code >= 400 or resp.data.get("statusCode") >= 400
```

Comment:
```suggestion
    return resp.status_code >= 400 or resp.data.get("statusCode", 0) >= 400
```

Otherwise this results in a `TypeError` trying to compare `None` with an `int`

Reply:
You know I was wondering about that but I figured the linter would have complained so I left it. Thanks for calling it out